service OneWay {
	oneway void hi(1: i64 i, 2: string s)
	void emptyfunc()
	i64 echo_int(1: i64 param)
}
